prdoc/pr_7720.prdoc

```diff
@@ -0,0 +1,18 @@
+# Schema: Polkadot SDK PRDoc Schema (prdoc) v1.0.0
+# See doc at https://raw.githubusercontent.com/paritytech/polkadot-sdk/master/prdoc/schema_user.json
+
+title: Clamp Core Fellowship Benchmarks to Runtime MaxRank Configuration
+
+doc:
+  - audience: Runtime Dev
+    description: |      
+      Enables reliable benchmarking for constrained configurations like `MaxRank=1` while
+      maintaining compatibility with larger rank ranges.
```

Comment:
Must mention the breaking change and how downstream developers integrate it.

Reply:
Done